__tests__/components/next_steps_test.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { NextSteps } from "../../components/next_steps";
+import { mount } from "enzyme";
+import translate from "../fixtures/translate";
+
+const { axe, toHaveNoViolations } = require("jest-axe");
+expect.extend(toHaveNoViolations);
+
+describe("NextSteps", () => {
+  let props;
+  let reduxState;
+  beforeEach(() => {
+    props = {
+      t: translate,
+      mapUrl: { query: {}, route: "/map" }
```

Comment:
This should actually just be a string - it's different from the url prop that is passed to each page, it's the result of the `getMapUrl` function in selectors/urls.js. That should fix your warning.

Reply:
But how do I make it a string and still the thing I want?

Reply:
Ok but then what do I put in the test for it on line 69? I think if I write "/map" then does the test still make sense??